entrypoint.sh

```diff
@@ -152,10 +152,10 @@ install_node()
 install_foundry()
 {
     if [[ -d "$TARGET" ]] && [[ -f "$TARGET/foundry.toml" ]]; then
-        echo "[-] Foundry target detected, installing foundry nightly"
+        echo "[-] Foundry target detected, installing foundry stable"
 
-        wget -q -O foundryup https://raw.githubusercontent.com/foundry-rs/foundry/7b452656f722fc560f0414db3ce24a1f2972a8b7/foundryup/foundryup
-        if [ ! "e7628766329e2873484d5d633c750b5019eec77ae506c11a0ef13b440cc3e7c2  foundryup" = "$(sha256sum foundryup)" ]; then
+        wget -q -O foundryup https://raw.githubusercontent.com/foundry-rs/foundry/41c6653e2adee0354ba6cdf7233c58294bcea3bf/foundryup/foundryup
```

Comment:
```suggestion
        wget -q -O foundryup https://raw.githubusercontent.com/foundry-rs/foundry/e144b82070619b6e10485c38734b4d4d45aebe04/foundryup/foundryup
```

Reply:
using a specific version could also work ... but IMO using a commit hash is not explicit enough.

Reply:
If you use a reference such as `stable` and the branch/tag is updated, all users will suddenly start failing the checksum though, which is undesirable.